assets/js/frontend/add-payment-method.js

```diff
@@ -32,4 +32,10 @@ jQuery( function( $ ) {
 
 	$( document.body ).trigger( 'init_add_payment_method' );
 
+	// Prevent firing multiple requests upon double clicking the buttons in payment methods table
```

Comment:
We can simplify to the following:

```javascript
$( 'the button selector here' ).on( 'click', function() {
		$(this).addClass('disabled');
});
```

- `addClass` IIRC will take care of not adding duplicate classes
- `the button selector here` could be `.woocommerce .payment-method-actions .button.delete` - for more flexibility and readability

Reply:
`preventDefault`, right, this might be needed. Please add it if it needed.

`I am happy to follow your suggestion`, delegation is an option, but IMO here we can take a simple approach, which is aligned with the rest of the file we are touching. Thanks !